src/transformers/pipelines/text_generation.py

```diff
@@ -242,7 +242,7 @@ def _forward(self, model_inputs, **generate_kwargs):
 
         # If there is a prefix, we may need to adjust the generation length. Do so without permanently modifying
         # generate_kwargs, as some of the parameterization may come from the initialization of the pipeline.
-        generate_kwargs = copy.deepcopy(generate_kwargs)
+        generate_kwargs = copy.copy(generate_kwargs)
```

Comment:
```suggestion
```

Even shorter version -- we don't need a copy at all, see the conversation [here](https://github.com/huggingface/transformers/issues/23785)